main.py

```diff
@@ -21,10 +21,22 @@
 # Init Database Connection
 client = MongoClient('mongodb://localhost', 27017)
 db = client["SmartRemote"]
+remote_collection = db["Remote"]
+
 
 # Main APIs
-@app.get("/")
-def main_api_age():
+@app.delete("/{{api_host}}/remote/{remote_id}}/")
+def delete_remote(remote_id: int):
+    query_remoteID = {
+      "remoteID": remote_id
+      }
+    query_result = remote_collection.find(query_remoteID, {"_id": 0})
+    remote_list = list(query_result)
+    if len(list_query_toilet) != 1:
+        raise HTTPException(400, {
+            "message": "The remote id was not found or there were 2 or more remotes found."
+        })
+    remote_collection.delete_one(query_remoteID)
```

Comment:
I think this API should remote the remoteId associated with the user. Not just the remote with specific ID